signing/secp256k1/api/src/main/java/tech/pegasys/signers/secp256k1/api/SignerProvider.java

```diff
@@ -20,6 +20,11 @@
 
   Optional<Signer> getSigner(ECPublicKey publicKey);
 
+  default Optional<Signer> getSigner(String address) {
+    // not applicable for Single Signer Provider mode
+    throw new UnsupportedOperationException("getSigner by address not supported");
```

Comment:
Think this should also work for the single signer provider, it's just an alternative to using the public key